src/project/src/test/scala/io/giovannini/EvenFibonnacciTest.scala

```diff
@@ -0,0 +1,24 @@
+package io.giovannini
```

Comment:
List(1) did not equal List()

Reply:
Comment ça se passe si le test est dans un fichier qui n'a pas été modifié ? C'est possible de faire un commentaire ?